pokemongo_bot/cell_workers/soft_ban_worker.py

```diff
@@ -0,0 +1,59 @@
+from pgoapi.utilities import f2i
+
+from pokemongo_bot import logger
+from pokemongo_bot.constants import Constants
+from pokemongo_bot.cell_workers import MoveToFortWorker, SeenFortWorker
+from pokemongo_bot.cell_workers.utils import distance
+from pokemongo_bot.worker_result import WorkerResult
+
+class SoftBanWorker(object):
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.api = bot.api
+        self.config = bot.config
+
+    def work(self):
+        if not self.should_run():
+            return
+
+        forts = self.bot.get_forts(order_by_distance=True)
+
+        if len(forts) == 0:
+            logger.log('Found no forts to reset softban, skipping...', 'red')
+            return
```

Comment:
The intent is that it would continue to the next worker if it doesn't have a fort available?